wagtailio/project_styleguide/templates/patterns/includes/header.html

```diff
@@ -5,14 +5,23 @@
         </svg>
     </a>
 
-    {% if get_started_menu_items %}
-        <button class="header__button header__button--get-started button" data-get-started>
-            Get Started
-            <svg class="modal__close-icon" aria-hidden="true">
-                <use xlink:href="#chevron"></use>
+    <div class="header__items-wrap">
+        <button class="header__button header__button--search" aria-label="Open the search" data-site-search>
+            <svg class="header__search-icon" aria-hidden="true">
```

Comment:
Comment defo think that this aria-hidden might not be correct, please shout if my understanding is wrong

Reply:
I think this one's ok based on the guidance from MDN and that the button has an `aria-label`